Add a "last updated at" timestamp in doc pages

fabcor-maxiv opened the pull request:
GitHub: https://github.com/mxcube/mxcubecore/issues/899

---

Preview: <https://mxcubecore--939.org.readthedocs.build/en/939/>

fabcor-maxiv committed:
Add a "last updated at" timestamp in doc pages

GitHub: https://github.com/mxcube/mxcubecore/issues/899

marcus-oscarsson commented:
Thanks :+1:

marcus-oscarsson commented:
@fabcor-maxiv I can't, for some reason, rebase this PR from GitHub can you rebase this and then merge ?

fabcor-maxiv commented:
@marcus-oscarsson 

> I can't, for some reason, rebase this PR from GitHub

Maybe because I made the PR from my own personal fork? I have no idea, but that could be a guess.


> can you rebase this and then merge ?

Just now I rebased and updated the PR. Or do you mean I should click the green "Rebase and merge" button in the GitHub UI?
